Thumbprint/Targets/ThumbprintTests/Snapshot/Shared/SnapshotTestCase.swift

```diff
@@ -82,7 +82,8 @@ open class SnapshotTestCase: XCTestCase {
         behavior = TestCaseBehavior()
     }
 
-    open override func tearDown() {
+    /// Override tearDownWithError as it gets called later by XCTestCase.
+    open override func tearDownWithError() throws {
```

Comment:
Why the need to migrate to `tearDownWithError()`?

Reply:
Oh this was a leftover from code churn, I can take it out if you want (we should probably start migrating to those newer methods anyway but that's unrelated to this PR).